src/reducers/index.ts

```diff
@@ -18,9 +42,84 @@ function reducer(state: Readonly<StateBase>, action: Action): StateBase {
   };
 }
 
+/**
+ * Exclude these actions from the history completely.
+ */
+export const ACTIONS_EXCLUDED_FROM_HISTORY: ActionType[] = [
```

Comment:
Actually, since `ACTIONS_EXCLUDED_FROM_HISTORY`, `USER_ACTIONS`, and `GROUPED_ACTIONS` are declarations about actions, maybe they should be in `src/actions/index.ts` more?